.github/workflows/Windows-CI.yml

```diff
@@ -0,0 +1,63 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Windows-CI
+
+# Controls when the action will run. 
+on:
+  ## Triggers the workflow on push or pull request events but only for the master branch
+  #push:
+  #  branches: [ master ]
+  #pull_request:
+  #  branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Set to run only manually at the moment, because the Windows build process takes so long (at least on my machine).

Reply:
That's fine while we debug this and time it.